ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst

```diff
@@ -0,0 +1,163 @@
+the lowest July temperatures? Now, double check your predictions by finding the
+mean temperatures of each of the three regions. You can do this by using the
+AVERAGEIF function :ref:`(from the section on grouping)<grouping_data>`.
+
+.. image:: figures/avg_if.png
+   :alt: A screenshot of a Sheet calculating the average Janurary temperatures of each region.
+
+Now that you have found the mean temperature answer the following questions:
+
+
+.. mchoice:: mean_temperature_jan
+
+   What was the mean January temperature for cities in the Central region?
+   (Rounded to the nearest whole number)
+
+   - 36
+
+     + Correct
+
+   - 20
+
+     - Incorrect
+
+   - 27
+
+     - Incorrect
+
+   - 33
+
+     - Incorrect
+
+
+.. mchoice:: mean_temperature_jul
+
+   What was the mean January temperature for cities in the Central region?
+   (Rounded to the nearest whole number)
+
+   - 75
+
+     + Correct
+
+   - 71
+
+     - Incorrect
+
+   - 68
+
+     - Incorrect
+
+   - 77
+
+     - Incorrect
+
+
+.. mchoice:: difference_mean
+
+   Which region has the largest difference between its January mean temperature
+   and its July mean temperature?
+
+   - Northern
+
+     + Correct
+
+   - Central
+
+     - Incorrect
+
+   - Southern
+
+     - Incorrect
+
+Functions like AVERAGEIF are a great way to calculate the mean for a specific
+region and/or month. But what if you want to see, in one visualization, the mean
+temperature for January and July across all regions? To do this, you can use a
+bar chart.
+
+
+.. image:: figures/avg_temp_region_jan.png
+   :alt: A bar chart depicting average Janurary and July temperatures for each of the regions.
+
+
+.. mchoice:: difference_regional
+
+   Looking at the visualization, which month has a bigger regional difference in
+   temperatures?
+
+   - Janurary
+
+     + Correct
+
+   - July
+
+     - Incorrect
+
+Although the above method is useful to determine the mean temperatures of
+different regions in the United States, it is not the most accurate since it
+applies over the whole region. For example, take the two cities Dayton, Ohio and
+Memphis, Tennessee. Both of these cities are in the Central region, but Dayton
+has a latitude of 39.54 degrees, while Memphis has a latitude of 35.07 degrees,
+which are quite different. Based on what you’ve learned, you could probably make
+an educated guess about which of these locations are colder in January, but how
+could you get a better prediction of what the temperature may be? This is where
+scatterplots will come in handy.
+
+
+.. _following dataset: https://drive.google.com/open?id=1Qhl09FAMxhHr9WVIKxfVK9UklvDVpiYMdgvKuaWZRa0
+.. _latitude, longitude: https://en.wikipedia.org/wiki/Geographic_coordinate_system
+.. See here for a map of the USA with latitude and longitude lines.: https://pasarelapr.com/images/map-of-usa-with-lines-of-latitude-and-longitude/map-of-usa-with-lines-of-latitude-and-longitude-2.jpg
```

Comment:
Do these references work in Runestone? Seems to me like they are missing something to make them work.